app_dart/bin/server.dart

```diff
@@ -82,29 +93,135 @@ Future<void> main() async {
         authProvider,
         scheduler,
       ),
+
+      /// Updates cocoon task status.
+      ///
+      /// This API updates task status in datastore.
+      ///
+      /// POST: /api-update-status
+      ///
+      /// Parameters:
+      ///   CommitBranch: (string in body). Branch of commit.
```

Comment:
In general, we should have defined RPC protos which would enforce documentation of requests and responses (this doesn't have to be done this PR). WDYT?

Reply:
Agreed, I am all for making the RPCs more defined.